src/database/DataTypes/MasterList.js

```diff
@@ -18,6 +18,35 @@ export class MasterList extends Realm.Object {
     if (this.items.filtered('id == $0', masterListItem.id).length > 0) return;
     this.addItem(masterListItem);
   }
+
+  getStoreTagObject(tags) {
+    const storeTags = tags.split(/[\s,]+/);
+    return Object.keys(JSON.parse(this.programSettings)).filter(
+      programsStoreTags => storeTags.indexOf(programsStoreTags) >= 0
```

Comment:
should be iterating through this.programSettings.storeTags

Reply:
thanks